anima/globals.css

```diff
@@ -0,0 +1,19 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css");
```

Comment:
Global comment: please remove ALL the Anima assets as they are not used in this implementation and might be subject to their own licensing and terms of use.